Cake.Core/Discord/Modules/CakeModule.cs

```diff
@@ -19,6 +22,70 @@ public CakeModule(CakeService service)
             _service.SetBaseModule(this);
         }
 
+        [Command("invite")]
+        [Summary("cake invite (isPermanentInvite?)")]
+        [Remarks("Fetches an invite URL to the current guild which this command is executed in.")]
+        public async Task GetGuildInvite(bool permanentInvite = true) {
+            if (!(Context.Guild is SocketGuild guild)) {
+                await ReplyAsync("Please use this command in a server only!");
+                return;
+            }
+
+            await SendInviteForUser();
+
+            #region Local_Function
+
+            async Task SendInviteForUser() {
+                RestInviteMetadata[] invites = await FetchInvitesByPermanentInvite();
+
+                if (invites.Length <= 0) {
+                    await ReplyAsync(GetErrorMessageToSend());
+                } else {
+                    var inviteToUse = invites.FirstOrDefault();
+                    await ReplyAsync(inviteToUse.Url);
+                }
+            }
+
+            string GetErrorMessageToSend() {
+                if (permanentInvite) {
+                    return $"There is no permanent invites in this server! {System.Environment.NewLine} Ask your owner to create one.";
```

Comment:
There is a space behind thje new line, the next line will be looking wierd in chat.

Could you please remove the space behind the new line.
Same goes for the other error message.

https://i.imgur.com/qCEFbzi.png

Reply:
Fixed, it.<br>Also, fixed newline indentation just to go with the supposed coding style in this document.